<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RedactingCarState</class>
 <widget class="QWidget" name="RedactingCarState">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>Редактировать состояние автомобля</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Добавить новый автомобиль</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Уникальный номер автомобиля</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLineEdit" name="uniqNumber"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Марка автомобиля</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="Type"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Состояние</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="state">
             <item>
              <property name="text">
               <string>Идеальное</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>С пробегом</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>На обслуживании</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Арендована</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Списана</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Количество топлива</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="fuel"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Пробег автомобиля</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="distance"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
